From 803667f7c8a8610b270ea0a50df9562b3e11025c Mon Sep 17 00:00:00 2001
From: mancha <mancha1@hush.com>
Date: Thu, 13 Feb 2014
Subject: Change hardcoded path for helper

SBo installs gksu-run-helper under /usr/lib64 for x86_64 systems.
This patch changes the hard-coded path so it is found.

 libgksu/libgksu.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/libgksu/libgksu.c
+++ b/libgksu/libgksu.c
@@ -1936,7 +1936,7 @@ gksu_su_fuller (GksuContext *context,
   GQuark gksu_quark;
   int i = 0;
 
-  gchar auxcommand[] = PREFIX "/lib/" PACKAGE "/gksu-run-helper";
+  gchar auxcommand[] = PREFIX "/lib64/" PACKAGE "/gksu-run-helper";
 
   int fdpty;
   pid_t pid;
